sequencer/src/bin/utils/keygen.rs

```diff
@@ -27,11 +27,11 @@ enum Scheme {
 }
 
 impl Scheme {
-    fn gen(self, seed: [u8; 32], index: u64, env_file: &mut impl Write) -> anyhow::Result<()> {
+    fn r#gen(self, seed: [u8; 32], index: u64, env_file: &mut impl Write) -> anyhow::Result<()> {
```

Comment:
maybe we rename this function to generate()?

Reply:
6fa1213b9a986a2ce53ad474bcaa2e452bffe45c